CGPs/cgp-0042.md

```diff
@@ -0,0 +1,90 @@
+# CGP [0042]: cBRL contract deploy
+
+- Date: 2021-10-09
+- Author(s): @luisgj @martinvol
+- Status: DRAFT
+- Governance Proposal ID #: 
+- Date Executed: TBD
+- Discussions To: [cBRL forum proposal](https://forum.celo.org/t/cbrl-stable-asset/1281)
+
+## Overview
+
+In this governance proposal, we propose to execute the Celo Core Contracts Upgrade (not a) Release 6, which mainly proposes the activation of the `StableTokenBRL` and `ExchangeBRL` smart contracts.
+
+TODO - Add link to release tag
+
+### Status
+- DRAFT = Feedback collection
+
+## Proposed Changes
+TODO - Gather Feedback on Proposed changes
```

Comment:
Gathering feedback should be done outside the specifications of the governance proposal.